src/schema/broadband-signup-us.json

```diff
@@ -0,0 +1,559 @@
+        },
+        "directDebit": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/DirectDebitIban"
+        },
+        "selectedTvSetupDate": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/SetupDate",
+            "sourceOutputKey": "availableTvSetupDates",
+            "inputMethod": "SelectOne"
+        },
+        "selectedBroadbandSetupDate": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/SetupDate",
+            "sourceOutputKey": "availableBroadbandSetupDates",
+            "inputMethod": "SelectOne"
+        },
+        "selectedActiveLandlineOption": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/String",
+            "sourceOutputKey": "availableActiveLandlineOptions",
+            "inputMethod": "SelectOne"
+        },
+        "selectedAddress": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/PIIString",
+            "sourceOutputKey": "availableAddresses",
+            "inputMethod": "SelectOne"
+        },
+        "securityInformation": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/Security"
+        },
+        "alternativeCorrespondence": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/NullableProperty"
+        },
+        "existingLandline": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/ExistingLandline"
+        }
+    },
+    "outputs": {
+        "availableMarketingContactOptions": {
+            "typeRef": "#/domains/Generic/types/StructuredTextItems",
+            "description": "List of marketing contact options provided by the insurer."
+        },
+        "availableTvPackages": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/Packages"
+        },
+        "availableBroadbandPackages": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/Packages"
+        },
+        "availablePhonePackages": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/Packages"
+        },
+        "availableAddresses": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/AvailableAddresses"
+        },
+        "availableTvSetupDates": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/AvailableSetupDatesOrNull"
+        },
+        "availableBroadbandSetupDates": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/AvailableSetupDatesOrNull"
+        },
+        "estimatedPriceBreakdown": {
+            "typeRef": "#/domains/Generic/types/StructuredTextItems"
+        },
+        "monthlyCosts": {
+            "typeRef": "#/domains/Generic/types/StructuredText"
+        },
+        "oneOffCosts": {
+            "typeRef": "#/domains/Generic/types/StructuredText"
+        },
+        "equipmentCosts": {
+            "typeRef": "#/domains/Generic/types/StructuredText"
+        },
+        "extrasCosts": {
+            "typeRef": "#/domains/Generic/types/StructuredText"
+        },
+        "upfrontPayment": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/Anything"
```

Comment:
Can you be more specific? `Anything` is too broad. Same for the ones below, please.